docs/specs/legacy.yml

```
@@ -326,7 +328,8 @@ inputs:
     monikerRange:
       'docs/v1/**': '< netcore-2.0'
       'docs/v2/**': 'netcore-2.0'
-    baseUrl: https://docs.com/docs
+    hostName: https://docs.com
+    basePath: docs
```

`basePath` in docs starts with `/` but does not ends with `/` unless it is `/`, can ensure that in `Config.OnDeserialized` method

```suggestion
    basePath: /docs
```